packages/integration/src/lib/catalog/catalog-browser-tool/catalog-browser-tool.component.ts

```diff
@@ -94,7 +94,9 @@ export class CatalogBrowserToolComponent implements OnInit, OnDestroy {
       });
 
     this.authenticate$$ = this.authService.authenticate$.subscribe(() => {
-      this.loadCatalogItems(this.catalog);
+      if (this.catalog) {
+        this.loadCatalogItems(this.catalog);
+      }
```

Comment:
On ne semble pas régler le problème mais plutot éviter de la renvoyer. Rapidement ça me semble être un problème de concurence entre les deux observables. Il faudrait trouver un moyen de corriger ce problème

Reply:
Oui exactement, problème de concurrence, parfois this.catalog est undefined.